_pkgdown.yml

```diff
@@ -68,8 +68,8 @@ navbar:
     articles:
       text: User Guides
       menu:
-      - text: Creating ADXX
-        href: articles/adxx.html
```

Comment:
Feel free to remove the actual files for this as well. It is located in the vingettes folder.